tests/unit/api/test_metadata_api.py

```diff
@@ -41,6 +41,24 @@
     "releaseTime": 123123,
 }
 
+MOCKED_DATASTRUCTURES = [
```

Comment:
Seems like the correct format for the mocked return would be:
```py
MOCKED_DATASTRUCTURES = {
        "INNTEKT_TJENPEN": {
            "description": "dummy",
            "operation": "ADD",
            "releaseTime": 123123,
        },
        "INNTEKT_BANKINNSK": {
            "description": "dummy",
            "operation": "ADD",
            "releaseTime": 123123,
        },
}
```

[source](https://github.com/statisticsnorway/microdata-metadata-service/blob/801b17edd421a8da795c7702ec5f5db512250caf/metadata_service/domain/metadata.py#L21)

Reply:
You're right, I fix 🏃